site/dotnet-api-guide.md

```diff
@@ -874,3 +875,84 @@ growing between recoveries.
 Acknowledgements with stale delivery tags will not be
 sent. Applications that use manual acknowledgements and automatic
 recovery must be capable of handling redeliveries.
+
+## <a id="oauth2-support" class="anchor" href="#oauth2-support">OAuth 2 Support</a>
+
+The client can authenticate against an OAuth 2 server like [UAA](https://github.com/cloudfoundry/uaa).
+The [OAuth 2 plugin](https://github.com/rabbitmq/rabbitmq-server/tree/main/deps/rabbitmq_auth_backend_oauth2)
+must be enabled on the server side and configured to use the same OAuth 2 server as the client.
+
```

Comment:
We are trying to move away from enable/disable in technical documentation to meet inclusive terminology IX OKRs, see the table here for words that need to be replaced: https://source.vmware.com/portal/pages/global-marketing/terminology-changes (search for "disable"), I suggest changing to "activated" or "turned on" which ever makes more sense technicaly